src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import classnames from 'classnames'
+
+type FooterProps = {
+  big?: boolean
+  medium?: boolean
+  slim?: boolean
+  /**
+   * Component for "return to top" button/handling
+   */
+  returnToTop?: React.ReactNode
+  /**
+   * Content in upper footer section, e.g. navigation, searchbar, signup form
+   */
+  primary: React.ReactNode
+  /**
+   * Content in lower footer section, e.g. contact information
+   */
+  secondary: React.ReactNode
+}
+
+// TODO: Add in "Return to Top" handling
+export const Footer = (
+  props: FooterProps & React.HTMLAttributes<HTMLElement>
+): React.ReactElement => {
+  const {
+    big,
+    medium,
+    slim,
+    returnToTop,
+    primary,
+    secondary,
+    ...footerAttributes
+  } = props
+
+  const classes = classnames(
+    'usa-footer',
+    {
+      'usa-footer--big': big,
+      'usa-footer--medium': medium,
+      'usa-footer--slim': slim,
+    },
+    footerAttributes.className
+  )
+
+  return (
+    <footer className={classes}>
```

Comment:
do we also want to include `{...footerAttributes}` here to pass along anything else?